simtools/model/camera.py

```diff
@@ -0,0 +1,670 @@
+import logging
+import numpy as np
+import matplotlib as mlp
+import matplotlib.pyplot as plt
+import matplotlib.patches as mpatches
+import matplotlib.colors as mcolors
+from matplotlib.collections import PatchCollection
+from scipy.spatial import cKDTree as KDTree
+
+import simtools.util.legend_handlers as legH
+from simtools.model.telescope_model import TelescopeModel
+from simtools.model.model_parameters import TWO_MIRROR_TELS, CAMERA_ROTATE_ANGLE
+
+__all__ = ['Camera']
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+
+class Camera:
+
```

Comment:
I understood we should also add docstrings for the class itself, despite of the description in the init. The attibutes list should go here, I guess.

Reply:
Yeah, I misunderstood the location. I now put the parameters in the init and the rest in the class definition. Not sure what to do with the general description, so I left it in both places. If you think it's OK, please resolve this conversation.

Reply:
Great. I also don't know where the general description is suppose to go. I assume it should go to the class, not to the init.